rom_data.py

```diff
@@ -386,9 +386,12 @@ def apply_layout_patches(rom: bytes) -> bytes:
     brinstar_top_clipdata = BackgroundTilemap.from_info(brinstar_top.clipdata, 117)
     brinstar_top_bg1 = BackgroundTilemap.from_info(brinstar_top.bg1, 287)
     brinstar_top_clipdata.set(0xE, 0x5, Clipdata.STEEP_SLOPE_BLTR, Clipdata.AIR)
-    brinstar_top_bg1.set(0xE, 0x5, 0x009A, 0x0106)
+    brinstar_top_bg1.set(0xE, 0x5, 0x009E, 0x0106)
+    brinstar_top_bg1.set(0xE, 0x6, 0x00AE, 0x00116)
     brinstar_top_clipdata.set(0xF, 0x4, Clipdata.STEEP_SLOPE_BLTR, Clipdata.SOLID)
-    brinstar_top_bg1.set(0xF, 0x4, 0x009A, 0x0092)
+    brinstar_top_bg1.set(0xF, 0x4, 0x009E, 0x0092)
+    brinstar_top_bg1.set(0xF, 0x5, 0x00AE, 0x00107)
+    brinstar_top_bg1.set(0xF, 0x6, 0x005F, 0x00117)
```

Comment:
I know the x and y positions for the tiles were hex before, but I'd appreciate if you'd change the first two args for all of these calls to decimal.
Anyway, tilemaps are an array of 16-bit numbers, so the latter two arguments should always be 4 hexits

Reply:
Looks good to me!